packages/nimble-components/src/accordion/specs/IxD.md

```diff
@@ -0,0 +1,193 @@
+
+<p>
+<kbd><img src="spec-images/Accordion-16.png"></kbd>
+<br>
+<sub>Disclosure with other content</sub>
+</p>
+
+### Tree & Navigation
+
+With a disclosure/accordion, the heading is a label for a group of content, not content in-and-of itself. With a tree, the content is the list items and all the list items represent data.
+
+| Component  | Content    | Organization |
+| ---------- | ---------- | ------------ |
+| Disclosure | Any        | Group labels |
+| Tree       | List items | Parent/child |
+
+[ARIA](https://www.w3.org/WAI/ARIA/apg/patterns/treeview/examples/treeview-navigation/) recommends avoiding using a tree component for navigation because the `tree` role requires unecessary implementation of complex functionality that is not needed for navigation styled to look like a tree with expandable sections. However, the accordion is not necessarily the correct component to try and replace it with.
+
+| Component  | Content    | Organization |
+| ---------- | ---------- | ------------ |
+| Disclosure | Any        | Group labels |
+| Tree       | List items | Hierarchical |
+| ???        | List items | Group labels |
+
+[**QUESTION:** Maybe a new navigation-specific component?]
+
+## Design
+
+### Interactions & States
+
+Header buttons are used to expand and collapse associated panels of content. The client user can specify which panels (if any) are open by default.
+
+<p>
+<kbd><img src="spec-images/Accordion-1.png"></kbd>
+<br>
+<sub>Default (all panels closed)</sub>
+</p>
+
+Hovering over a header button indicates interactivity.
+
+<p>
+<kbd><img src="spec-images/Accordion-2.png"></kbd>
+<br>
+<sub>Hover</sub>
+</p>
+
+Clicking anywhere on a header button opens (if closed) or closes (if opened) the associated panel of content. [**QUESTION:** Do panels only display block content (one component per "line") or do they display flex content?]
+
+<p>
+<kbd><img src="spec-images/Accordion-3.png"></kbd>
+<br>
+<sub>Click</sub>
+</p>
+
+The client user can specify whether one panel or more than one panel can be opened at a time. If the former, clicking to open a closed panel will close the open panel (if applicable).
+
+<p>
+<kbd><img src="spec-images/Accordion-4.png"></kbd>
+<br>
+<sub>Click (one panel open at a time)</sub>
+</p>
+
+If a component in a panel is in an error state, the error state is indicated on the associated header button when closed (if not also when open).
+
+<p>
+<kbd><img src="spec-images/Accordion-5.png"></kbd>
+<br>
+<sub>Error</sub>
+</p>
+
+### Additional Options
+
+Client users may need/want to have additional buttons and/or icons in the header.
+
+<p>
+<kbd><img src="spec-images/Accordion-6.png"></kbd>
+<br>
+<sub>"Expand" button</sub>
+</p>
+
+### Nested Accordions
+
+Accordion panels can have one level of nested accordions. Nested accordions are always ghost style. [**QUESTION:** Can accordions be nested anywhere in the panel? Can there be multiple child accordions nested in an accordion?]
```

Comment:
I didn't include it because it felt like a visual design thing. One thing I learned from this first spec was that I think I need to reach out to designers (if not also clients?) to get their cases/context for the component, which could help inform what visual styles are needed. But I don't feel like it's an interaction design decision. @NIbokeefe thoughs?